src/Filter.js

```diff
@@ -120,8 +118,8 @@ class Filter extends Component {
               return (
                 <li className="country-list">
                   {" "}
-                  <span className="country-name"> {country.name} </span>{" "}
-                  population: {country.population}{" "}
+                  <span className="country-name"> {country.name} </span>
```

Comment:
this is weird, i think the vs code linting added this

Reply:
Agreed

Reply:
@jamesb8891 is your linter acting wierd?